src/control/server/ctl_storage_rpc.go

```diff
@@ -167,16 +169,20 @@ func (c *StorageControlService) StorageScan(ctx context.Context, req *ctlpb.Stor
 		}
 	}
 
-	modules, namespaces, err := c.ScanScm()
+	result, err := c.ScanScm()
```

Comment:
ScanScm appears to have been removed in this changeset... Should just use Discover, right?